papaparse.js

```diff
@@ -415,6 +415,24 @@
 		}
 	}
 
+	function DynamicTypingConfig(config)
```

Comment:
Still no need for a class. Just allow a user function and if no function is provided use the current one with the config object.